mypyc/emitmodule.py

```diff
@@ -117,10 +117,18 @@ def generate_c_for_modules(self) -> str:
             self.declare_module(module_name, emitter)
             self.declare_internal_globals(module_name, emitter)
             self.declare_imports(module.imports, emitter)
-            self.declare_finals(module.final_names, emitter)
 
-        for identifier in self.literals.values():
-            self.declare_static_pyobject(identifier, emitter)
+        for (_, literal), identifier in self.literals.items():
+            if isinstance(literal, int):
+                symbol = emitter.static_name(identifier, None)
+                self.declare_global('CPyTagged ', symbol)
+                self.declare_global('PyObject *', INT_PREFIX + symbol)
```

Comment:
Probably we don't want to make the boxed version be a global? If we need a local variable during initialization it should just stay a local.

Reply:
Indeed. Will fix this.